suite-common/graph/src/graphDataFetching.ts

```diff
@@ -163,24 +235,26 @@ const fiatRatesCache: Record<string, FiatRatesItem[]> = {};
 export const getFiatRatesForNetworkInTimeFrame = async ({
     timestamps,
     coin,
+    contractId,
     fiatCurrency,
     forceRefetch,
     isElectrumBackend,
 }: {
     timestamps: number[];
     coin: NetworkSymbol;
+    contractId?: TokenAddress;
     fiatCurrency: FiatCurrencyCode;
     forceRefetch?: boolean;
     isElectrumBackend: boolean;
 }) => {
-    const cacheKey = `${coin}-${fiatCurrency}-${JSON.stringify(timestamps)}`;
+    const cacheKey = `${coin}-${contractId}-${fiatCurrency}-${JSON.stringify(timestamps)}`;
```

Comment:
contractId is possibly undefined, is it ok here?

Reply:
No issue at all, it will make unique string either way which is what we want here.

Reply:
👌